pandas/tests/plotting/test_series.py

```diff
@@ -818,6 +818,9 @@ def test_xlabel_ylabel_series(self, kind, index_name, old_label, new_label):
         if kind == "barh":
             assert ax.get_xlabel() == ""
             assert ax.get_ylabel() == old_label
+        elif kind == "hist":
```

Comment:
I think we need a separate test to check if a user passes a custom xlabel and ylabel

Reply:
I believe this is covered with lines 829-831, which are not showing up in the diff without expanding it.
<img width="736" alt="Screen Shot 2022-12-03 at 2 27 10 PM" src="https://user-images.githubusercontent.com/24376333/205458400-94962461-382c-4f55-94c1-cde91b718765.png">

Reply:
Ah gotcha, thanks!